blockly-core/README.md

```diff
@@ -1,7 +1,29 @@
-# Blockly Core
+# Blockly Core (Code Studio fork)
 
 This is a fork of [Blockly](https://code.google.com/p/blockly/), an open source visual programming environment.
 
+Major additions and changes in this fork:
```

Comment:
@joshlory @Bjvanminnen @breville @islemaster @Hamms any other major features/changes worth mentioning in this fork description section?